price-feeder/config/config.go

```diff
@@ -60,92 +59,92 @@ var (
+		WriteTimeout   string   `mapstructure:"write_timeout"`
+		ReadTimeout    string   `mapstructure:"read_timeout"`
+		VerboseCORS    bool     `mapstructure:"verbose_cors"`
+		AllowedOrigins []string `mapstructure:"allowed_origins"`
 	}
 
 	// CurrencyPair defines a price quote of the exchange rate for two different
 	// currencies and the supported providers for getting the exchange rate.
 	CurrencyPair struct {
-		Base      string          `toml:"base" validate:"required"`
-		Quote     string          `toml:"quote" validate:"required"`
-		Providers []provider.Name `toml:"providers" validate:"required,gt=0,dive,required"`
+		Base      string          `mapstructure:"base" validate:"required"`
+		Quote     string          `mapstructure:"quote" validate:"required"`
+		Providers []provider.Name `mapstructure:"providers" validate:"required,gt=0,dive,required"`
 	}
 
 	// Deviation defines a maximum amount of standard deviations that a given asset can
 	// be from the median without being filtered out before voting.
 	Deviation struct {
-		Base      string `toml:"base" validate:"required"`
-		Threshold string `toml:"threshold" validate:"required"`
+		Base      string `mapstructure:"base" validate:"required"`
+		Threshold string `mapstructure:"threshold" validate:"required"`
 	}
 
 	// Account defines account related configuration that is related to the Umee
 	// network and transaction signing functionality.
 	Account struct {
-		ChainID   string `toml:"chain_id" validate:"required"`
-		Address   string `toml:"address" validate:"required"`
-		Validator string `toml:"validator" validate:"required"`
+		ChainID   string `mapstructure:"chain_id" validate:"required"`
+		Address   string `mapstructure:"address" validate:"required"`
+		Validator string `mapstructure:"validator" validate:"required"`
 	}
 
 	// Keyring defines the required Umee keyring configuration.
 	Keyring struct {
-		Backend string `toml:"backend" validate:"required"`
-		Dir     string `toml:"dir" validate:"required"`
+		Backend string `mapstructure:"backend" validate:"required"`
+		Dir     string `mapstructure:"dir" validate:"required"`
 	}
 
 	// RPC defines RPC configuration of both the Umee gRPC and Tendermint nodes.
 	RPC struct {
-		TMRPCEndpoint string `toml:"tmrpc_endpoint" validate:"required"`
-		GRPCEndpoint  string `toml:"grpc_endpoint" validate:"required"`
-		RPCTimeout    string `toml:"rpc_timeout" validate:"required"`
+		TMRPCEndpoint string `mapstructure:"tmrpc_endpoint" validate:"required"`
+		GRPCEndpoint  string `mapstructure:"grpc_endpoint" validate:"required"`
+		RPCTimeout    string `mapstructure:"rpc_timeout" validate:"required"`
 	}
 
 	// Telemetry defines the configuration options for application telemetry.
 	Telemetry struct {
 		// Prefixed with keys to separate services
-		ServiceName string `toml:"service_name" mapstructure:"service-name"`
+		ServiceName string `mapstructure:"service_name" mapstructure:"service-name"`
```

Comment:
Let's see if we can use the cosmos sdk's telemetry config struct instead. Ref: https://github.com/umee-network/umee/pull/1121/files#r918038413